enrolments/tests/serializers/test_session_create.py

```diff
@@ -0,0 +1,54 @@
+from django.core.exceptions import ValidationError
+from django.test import TestCase
+from unittest.mock import patch
+
+from accounts.models import User
+from enrolments.models import Class, Session
+from enrolments.serializers import SessionCreateSerializer
+from datetime import date
+
+
+class SessionCreateSerializerTestCase(TestCase):
+    def setUp(self):
+        self.name = "Fall 2020"
+        self.start_date = date(2020, 9, 5)
+        self.fields = [1, 2, 3]
+        self.facilitator = User.objects.create(email="user@staff.com")
+
+        # self.class1 = Class.objects.create(
+        #     name="Test Class 1",
+        #     days=[Class.TUESDAY, Class.SATURDAY],
+        #     location="219 Waterloo Way",
+        #     facilitator=self.facilitator.id,
+        # )
+        # self.class2 = Class.objects.create(
+        #     name="Test Class 2",
+        #     days=[Class.MONDAY, Class.THURSDAY],
+        #     location="1378 Kitchener Street",
+        #     facilitator=self.facilitator.id,
+        # )
```

Comment:
Whenever I call create(), it throws the OG `Class.facilitator" must be a "User" instance` error :-[
I was able to avoid this by just nesting this in the session payload, but not sure if this is what we wanna do 🤔

Reply:
I think here you'd need to either do `facilitator_id=self.facilitator.id` or `facilitator=self.facilitator`!